root.go

```diff
@@ -65,5 +68,13 @@ func app() {
 		negroni.Wrap(crudRoutes),
 	))
 	n.UseHandler(r)
-	n.Run(fmt.Sprintf(":%v", config.PrestConf.HTTPPort))
```

Comment:
`.Run` has been removed, where will it be invoked?

Reply:
On func startServer